### ISSUE TITLE
Writing to disk is not persisted to adf 

### ISSUE THREAD

**Issue by simontoens** [emufreak/iAmiga#47]
I am seeing the following behavior using Workbench/CLI:
1) Boot into Workbench
2) Insert another disk into df1:
3) Copy a file to df1: such as "Clock"
4) Close the df1: window and re-open it - "Clock" is there
5) Eject df1: and re-insert the disk - "Clock" is gone.

It seems like the modifications are made in memory only, but not written to the underlying adf file.

**Comment by mithrendal** [comment]
Isn't it possible to write back on eject or emulator exit? Do you mean eject from iuae menu?

**Comment by simontoens** [comment]
@mithrendal yes "eject" from the iuae disk menu.  

I think there is an option in the core emulator to enable writing back modifications to the underlying adf file, at least I saw this in fs-uae.  So my guess is that it is just a matter of finding and exposing that setting in the iuae disk setting.

**Comment by mithrendal** [comment]
If I need to write back persistently I use for now your save states feature, this is at least some sort of a workaround for this issue. Better would be of course a ro/rw setting as you did with harddrives...

**Comment by simontoens** [comment]
Yes I don't think this is very high priority.  

In my opinion, now that you fixed the hires issue, the next most important issue that needs to be fixed is the one about some games running too fast, specifically all the Cinemaware games I have tried...

**Comment by mithrendal** [comment]
What about #62 ? Maybe I am lucky and can find it in a short time. That's about it all. Fixing is not that difficult. But finding is much harder as it implies that one understands the existing code and underlying concepts. That was also the case with the hires issue. 

Also what I like to do is set vertical stretch automatically. When you take fs uae for example, it always scales the height automatically.

**Comment by simontoens** [comment]
Yes agreed #62 is a good one, specially since you are already in that area of the code...

Yes, finding is super hard!

**Comment by emufreak** [comment]
Just noticed that writing to df0 doesnt work anymore also. This makes it high priority in my opinion as it makes games relying on savestates unusable.

**Comment by simontoens** [comment]
@emufreak as far as I know writing to any floppy disk drive has never worked.  But it sounds like you think it has worked before?  As far as I remember I have always used savestates to persist floppy state.

**Comment by emufreak** [comment]
@simontoens I know it worked on early 1.0 releases. I used it to play Lemmings 2 back then. Didn't use it anymore since then however

**Comment by simontoens** [comment]
@emufreak ok good to know.  Personally I think the workaround @mithrendal suggested, saving emulator states, is still valid.  But yeah, we should fix this...
